src/main/frontend/components/block.cljs

```diff
@@ -530,27 +530,28 @@
 (defn open-page-ref
   [e page-name redirect-page-name page-name-in-block contents-page? whiteboard-page?]
   (util/stop e)
-  (cond
-    (gobj/get e "shiftKey")
-    (when-let [page-entity (db/entity [:block/name redirect-page-name])]
-      (state/sidebar-add-block!
-       (state/get-current-repo)
-       (:db/id page-entity)
-       :page))
+  (when (not (util/right-click? e))
```

Comment:
This will disable right clicking on refs globally. Please confirm that this is OK.

Reply:
Which problem does this change address?
Right-clicking on refs could be used later for configuring the page's schema.